[UPDATE] scrolling and pagination coming screen to new empty state (#5472)
grommet/hpe-design-system: aries-site/src/pages/templates/scrolling-and-pagination.mdx

import { EmptyState } from 'aries-core'
import { InProgress } from '@hpe-design/icons-grommet'

<ScrollingAndPaginationEmptyState />

export const ScrollingAndPaginationEmptyState = () => (
  <EmptyState
    level={2}
    title="Update in progress"

>     description="This page is temporarily unavailable. Content is being prepared and will be available soon."

Reviewer: ```suggestion
    description="This page is unavailable. Content will be available in the future."
```
